modules/azure/src/main/java/org/testcontainers/containers/AzuriteConnectionStringBuilder.java

```diff
@@ -0,0 +1,93 @@
+package org.testcontainers.containers;
+
+import org.rnorth.ducttape.Preconditions;
+
+/**
+ * Builds connection strings for the {@link AzuriteContainer}.
+ */
+class AzuriteConnectionStringBuilder {
```

Comment:
this class duplicates a lot from Azuritecontainer already. Let's keep it in that class.

Reply:
Done, thank you!